test/topology_tasks/suite.yaml

```diff
@@ -1,12 +0,0 @@
-type: Topology
-pool_size: 10
-cluster:
-  initial_size: 0
-extra_scylla_config_options:
-    authenticator: AllowAllAuthenticator
-    authorizer: AllowAllAuthorizer
-    enable_user_defined_functions: False
-    experimental_features: ['consistent-topology-changes']
-    enable_tablets: True
-extra_scylla_cmdline_options:
-  - "--task-ttl-in-seconds=10000000"
```

Comment:
I think some of these options are not in test/topology_custom. Did you check or think that it's fine to not have them?